data-serving/scripts/convert_data.py

```diff
@@ -38,14 +46,35 @@ def read_csv(infile: str) -> DataFrame:
     return pd.read_csv(infile, header=0, low_memory=False, encoding='utf-8')
 
 
-def convert(original_cases: DataFrame) -> DataFrame:
-    logging.info('Converting %d cases', len(original_cases))
-    # TODO: Much, much more conversion.
-    return original_cases.rename(columns={"ID": "_id"})
+def convert(cases: DataFrame) -> DataFrame:
+    logging.info('Converting %d cases', len(cases))
+
+    # [[_id]]
+    cases = cases.rename(columns={'ID': '_id'})
+
+    # [[demographics]]
+    cases['demographics'] = cases.apply(
+        lambda x: convert_demographics(x['_id'], x['age'], x['sex']), axis=1)
+
+    # [[events]]
+    cases['events'] = cases.apply(
+        lambda x: convert_events(x['_id'], {
+            'onsetSymptoms': x['date_onset_symptoms'],
+            'admissionHospital': x['date_admission_hospital'],
+            'confirmed': x['date_confirmation'],
+            'deathOrDischarge': x['date_death_or_discharge']
+        }, x['outcome']), axis=1)
+
+    # Filter out deprecated columns
+    cases = cases.filter(
+        items=['demographics', 'events'])
```

Comment:
ah, yes, it does. _id is going to be auto-generated by mongodb on import instead -- the ids used in the original data don't work as ObjectIds (not valid hexes). in the next PR i'm sending, it will be archived in importedCase, though.

removing the line where we rename it from ID to _id since that's useless now